### ISSUE TITLE
Decide on compensation per BSQ for locked up for bonded roles

### ISSUE THREAD

**Issue by chimp1984** [bisq-network/proposals#120]
Bonds locked up **for bonded roles** should be compensated by <strike>a kind of interest rate</strike> a compensation for each locked up BSQ per cycle to reflect the volatility risk for locking up a bond for long time as well as the risk for confiscation. 
We should find a consensus what we consider a fair % rate. The rate would be paid until a role gets revoked and **would be simply added to the role owners comensation request (no automated payment)**.

I would suggest 0.5-1 % per cycle (roughly month), but no strong opinions.
Please post below your suggestions so that we find a common ground.

UPDATE:
Changed the misleading term interest rate to compensation for the amount fo BSQ locked up per time interval. Also added emphasis that this is part of a normal compensation request and no automated process where BSQ earn a interest rate just by locking them up.

This proposal is only for bonded roles not for reputation bonds or future usage of bonds for the off-chain trade protocol.

**Comment by m52go** [comment]
TLDR; I think a 1% rate is reasonable. I write the following to attempt a reasoning.

This is such an odd scenario that I find it hard to determine an answer based on anything from the real world. Perpetual bonds are probably the closest thing but still very different in nature and purpose.

There is effectively no term for this "loan". The supplier of capital is on the hook for their _own_ performance, unlike a typical loan/bond where the supplier of capital (bank, investor, etc) is trusting the other party to perform (consumer, company, etc). In these cases the rate can be derived from various attributes of that party and the length of the term. None of these conventions are applicable here.

There is currently 130 000 BSQ locked up, and it looks like about 400 000 [remains to be locked up](https://github.com/bisq-network/bisq/blob/fc72225fa350c13e0ba1cf4849cdbdee3fc2fb31/core/src/main/java/bisq/core/dao/state/model/governance/BondedRoleType.java) (not including arbitrators). So say total bonds for the network end up between 500 000 and 1 000 000 in the long term. 

At 1% per month, the network would be paying 10 000 BSQ in interest per month, the approximate cost of one contributor. In return, the network can maintain reasonable confidence that all of its most valued functions and assets are being handled responsibly, as well as the capability for recourse in the event something goes wrong. Remember this assumes a 1 000 000 total, so for now this number will be much lower, probably in the 1 000 to 5 000 range.

This seems reasonable to me. If anything, this is a low rate for the value it provides. But at 12% annualized return for bondholders, I would probably not be in favor of anything higher.

**Comment by bodymindarts** [comment]
I don't see why there has to be an interest. Locking up allows you to perform a role which you get paid for. This could be viewed as a form of interest already. This is where the value is being added.

**Comment by mpolavieja** [comment]
> 1. maintaining a BSQ bond

@m52go I think that your suggestion of structuring it as a "*kind of* dividend" is fair.   The fact that locked BSQ accrues compensation and not locked BSQ accrues nothing fully complies with the philosophy that only valuable contributions should be compensated.   A good way to know if a  contribution that involves no work or very little work is valuable is through profit (ie BSQ burnt > BSQ issued).

**Comment by m52go** [comment]
> The fact that locked BSQ accrues compensation and not locked BSQ accrues nothing fully complies with the philosophy that only valuable contributions should be compensated.

@mpolavieja that's an interesting way of thinking about it.

To me the appeal of a dividend is that it would apply to all BSQ holders, which addresses the concerns of those in this thread against "interest" for a small group of people, while still achieving the goal of this proposal.

While I still like this idea in theory, I dismissed it because I don't see any practical way it could work, as it would require every BSQ holder to make a compensation request. I don't believe the DAO is designed to handle so many requests, and furthermore, I fear such a mechanism would change BSQ's public perception from being a utility token to some kind of investment. 

I think the problem at hand is solvable by easier means, such as figuring out a way to compensate for ongoing work. It's a more solid justification for compensation, while indirectly providing a return for locked-up funds.

**Comment by chimp1984** [comment]
Burning BSQ is conceptually a divident payment to all BSQ stakeholder. Each unit gets a bit more valuable if the total supply is lower due burnt BSQ. So each single fee payment by burning BSQ is a dividened payment to all BSQ stakeholders.

Beside that locking up BSQ removed available supply as well for the time the bond is locked up. So each bond holder creates a similar effect like burning at lockup time. At unlock time though it behaves more like an issuance (increasing supply my making BSQ available again).

**Comment by chimp1984** [comment]
@wiz 
> Is this already implemented in the DAO ? If not, I don't think it's worth the risk of hard-forking the DAO to add such a feature. 

It was not intended to make that automatic, but just to find a consensus what is a fair compensation for bonded BSQ so that all use the same calculation based on their bond value.

I think there is a confusion triggered by the term "interest rate".

All I suggest is to find a conesensus what bond holders should request as compensation for the service to be willing to lock up BSQ (and having the risk to get confiacated as well to be not able to react on volatility for trading). 
As noted, there are some roles where basically no effort is done (domain holder, dns,...) and beside that it would be good to have it transparent how much of a compensation for a normal role is for the bond.

E.g. a seed node operator might spend 80 EUR for hosting costs. He might take 100 EUR for his service to maintain and operate the node. He has to lockup 10 000 BSQ and he wants to get some compenastion for that as well. If we do not agree on a consensus, operator A might charge 10 BSQ and operator B might charge 100 BSQ which would trigger discussions.... 
Btw. hosting costs should be aligned as well so that there is not huge differences (but some differences are ok) - but that is another topic....

**Comment by mpolavieja** [comment]
A dividend does not necessarily need to accrue to all BSQs.  Like different classes of stock issuance where not all classes have the same rights.   In any case I would not call it a dividend but a compensation for specific roles that is calculated in a specific way (a % of BSQ burnt - BSQ issued per cycle)

**Comment by chimp1984** [comment]
I renamed the title to make it more clear and avoid that "interest rate" confusion. Also edited the text to make it more clear.

If still too confusing or too much problems with that, I am fine with closing it as rejected and leave it to others to find a suggestion how to deal with it.

**Comment by MwithM** [comment]
There's risk involving the tasks you are responsible for, which is different between roles and then there's risk involving cost of oportunity of locked funds and being "forced" to leave  funds in a new and possibly volatile asset. This kind of risk is the same for all those who have their funds locked in bonds, even though they might estimate it in a different way because there might be contributors who feel more or less confident on the future of BSQ.
I think that removing "interest rate" from the subject of this issue is correct, we just need to find a price for locked bonds. It's normal we don't find a price easily because we need to look to the future to value risks and possible value of this project, and there's not many locked bonds already to compare between different roles. 
Claiming a compensation request is a good way to pay to those willing to risk their funds, as funds are substracted from the DAO in form of inflation.
For the future, if there are ennough trusted contributors interested, maybe we could stablish a periodical auction for bonding price to fulfill the roles needed by the DAO.

**Comment by bodymindarts** [comment]
2 questions:
- We are referring to bonds that relate to roles right? Not _all_ bonds should receive compensation (ie. reputation, and later perhaps deposit for trading).
- Is it necessary to come to global consensus about this? What is wrong with each person making their own decision when they submit their request? Wouldn't that be more of a free and decentralized market approach?

**Comment by chimp1984** [comment]
@bodymindarts 
We refer to bonds from bonded roles only, not reputation bonds or those used in the off-chain trade protocol. I will add the "bonded role" part again to the title and the introduction.

It is not necessary and we did not had consensus yet and bonded roles are already used. But it would lead to conflicts if people use different rates as it should be the same for all. Why should one seed node operator charge 100 BSQ and another 50 BSQ for a 10 000 BSQ bond. 

Bondes roles are not designed to be picked up the market. They are given to people who have already earned reputation and trust. The confiscation part is just an extra security added to that based on trust and reputation. Also they are not intended to be changed frequently which migth be problematic for some (like nodes where onion address is hard coded).

**Comment by m52go** [comment]
Another way of thinking of locked-up bond is that it's a commitment to consistently do some quantity of non-deliverable work, as the bond is confiscated if this work isn't done. 

Whether this compensation is calculated as a percentage of locked bond, or as a nominal BSQ amount per role based on estimated monthly commitment...that seems to be the main question here.

**Upsides/downsides**

_Percentage of bond_
Ongoing compensation will not relate to effort. For example, Slack admin and seednode operators both post 20 000 BSQ bonds, but Slack requires very little work (if any) on a regular basis.

_Estimated monthly commitment_
Compensation is more closely tied to performed work, but it's not related to posted bond. Domain name roles lose out here, for example, but I would expect the network to be willing to make reasonable exceptions for such cases.

The second case seems to be more favored here, of course, but I wanted to articulate this to show that the first case shouldn't seem too far from the second one.

**Comment by chimp1984** [comment]
> Obviously the second case seems to be more favored here

Seems so, but not sure if it is caused mainly by misunderstandings (not clear if for bonded roles or for all bonds, etc.).

Maybe we should just close that now and once it becomes a real-life problem (different bond holders will request different % amounts) anyone can give it another try. Does not make sense to burn so much time/energy for such a minor topic.
